README.rst

```diff
@@ -29,15 +29,33 @@ Usage:
 
     jinja2_standalone_compiler <path-to-jinja-files>  # Note that jinja files will be searched recursively
 
-For a more intense usage, you can also use a settings file to define a bunch of things. To use settings file:
+* For a more intense usage, you can also use a settings file to define a bunch of things. To use settings file:
 
-.. code:: bash
+    .. code:: bash
+
+        jinja2_standalone_compiler <path-to-jinja-files> --settings=path/to/settings.py
+
+    Please note that settings file is a Python file, so Python syntax applies. =P
```

Comment:
In fact, it doesn't look like code.  :laughing: 

I'll use nested bullets here instead. Just tested on git and rendered as it should.
I'll make this change.